### PR TITLE
Add Nix build: 2.4pre20210317_8a5203d

### PR DESCRIPTION
**Pull request by tadfisher**
Update to the latest `nixUnstable` as some commands have changed syntax.

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -61,6 +62,7 @@
       ) [
         pkgs.nixUnstable
         nixpkgs-nix-unstable-20200829.legacyPackages.${system}.nixUnstable
+        nixpkgs-nix-unstable-20200829.legacyPackages.${system}.nixUnstable
```
Comment (rickynils) [inline flake.nix:65]: I think this was meant to be `nixpkgs-nix-unstable-20201205` instead?

Comment (tadfisher) [reply flake.nix:65]: Nice catch, thanks!

**Comment by rickynils** [comment]
@tadfisher It seems like the MacOS build doesn't work. Maybe bumping nixpkgs to include https://github.com/NixOS/nixpkgs/pull/117758 solves it? I can try it out later this week if you don't beat me to it.

**Comment by rickynils** [comment]
Closing this, because Nix 2.4pre20210601_5985b8b has now been added.

**Pull request closed without merging**